src/main/java/fr/unice/polytech/si3/miaou/brainfuck/parser/InstructionTextParser.java

```diff
@@ -36,6 +36,9 @@ public void accept(String line) {
 			addAndBindInstructions(instr);
 		} else if (iset.getProc(split[0]) != null) {
 			Procedure proc = iset.getProc(split[0]);
+			if (split.length > 1) {
```

Comment:
The whole `else if (iset.getProc(split[0]) != null)` should probably be done in a method of (let's say) FunctionParser.